src/sentry/install.js

```diff
@@ -11,6 +13,60 @@ export default (Raven: any, shouldSendCallback: () => boolean, userId: string) =
     tags: { git_commit: __GIT_REVISION__ },
     environment: __DEV__ ? 'development' : 'production',
     shouldSendCallback,
+    dataCallback: (data: mixed) => {
```

Comment:
:sunglasses: :+1: :rocket:

Reply:
this is annoying mode of Flow but at least safe. I was looking at alternative like `lodash update` but i have no experience in it and no idea if it's safe

Reply:
yeah. totally agree on using those kind of vanilla/quickly fixable helpers for any sensitive part